readme.md

```diff
@@ -119,6 +119,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 ## Development
 
 - [bcal](https://github.com/jarun/bcal) - Byte CALculator for storage conversions and calculations.
+- [bitwise](https://github.com/mellowcandle/bitwise) - Terminal based bit manipulator in ncurses.
```

Comment:
As stated in the [contribution guidelines](https://github.com/agarrharr/awesome-cli-apps/blob/master/contributing.md#contribution-guidelines):
**Additions should be added to the bottom of the relevant category.**

Reply:
Could you also strip down the description to remove unnecessary information:
E.g.: `Terminal based bit manipulator in ncurses.` ->
`Bit manipulator in ncurses.`
This list is about clis, being terminal based is a given :)